sdk/loadtestservice/Azure.Developer.LoadTesting/samples/Sample1_CreateOrUpdateTest.md

```diff
@@ -0,0 +1,56 @@
+# Create or Update Load Test 
+
+To use these samples, you'll first need to set up resources. See [getting started](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/loadtestservice/Azure.Developer.LoadTesting/README.md#getting-started) for details.
+
+You can create a LoadTestclient and call the `CreateOrUpdateTest` method from SubClient `LoadTestAdministrationClient`
+
+## Create LoadTestAdministrationClient
```

Comment:
Change this to "Create LoadTesting"

Reply:
@ninallam idea here to show the sample of using correct sub-client, do we want to change it to "LoadTesting" from "LoadTestAdministrationClient"